tests/meson.build

```diff
@@ -100,7 +100,8 @@ endif
 unittest_tests_install_dir = join_paths(unittest_install_dir,'tests')
 tensor_filter_ext_enabled = get_option('enable-tensorflow-lite') or
     get_option('enable-python') or  get_option('enable-tensorflow') or
-    get_option('enable-pytorch') or get_option('enable-caffe2')
+    get_option('enable-pytorch') or get_option('enable-caffe2') or
+    get_option('enable-nnfw-runtime')
```

Comment:
This meson script might be not compatible with 0.51.
It didn't emit error messages in < 0.51, but that happened to be a bug.
You'd probably need ```\``` at the end of these lines.

Reply:
Updated.